Tasks/AzureFunctionV2/task.json

```diff
@@ -0,0 +1,156 @@
+      "defaultValue": "ARMAuthentication",
+      "options": {
+        "KeyAuthentication": "Function Key",
+        "ARMAuthentication": "Azure Resource Manager"
+      }
+    },
+    {
+      "name": "ARMAuthentication",
+      "aliases": ["azureServiceConnection", "azureSubscription"],
+      "type": "connectedService:AzureRM",
+      "label": "Azure subscription",
+      "required": true,
+      "helpMarkDown": "Select an Azure Resource Manager subscription to use for invoking a management API. 'ManagementPortalUrl' of the cloud is used as the baseURL",
+      "visibleRule": "authenticationOptionSelector = ARMAuthentication"
+    },
+    {
+      "name": "function",
+      "type": "string",
+      "label": "Azure function URL",
+      "defaultValue": "",
+      "required": true,
+      "helpMarkDown": "URL of the Azure function that needs to be invoked​. Example:- https://azurefunctionapp.azurewebsites.net/api/HttpTriggerJS1"
+    },
+    {
+      "name": "key",
+      "type": "string",
+      "label": "Function key",
+      "defaultValue": "",
+      "required": true,
+      "helpMarkDown": "Function or Host key with which to access this function. To keep the key secure, define a secret variable and use it here. Example: - $(myFunctionKey) where myFunctionKey is a secret pipeline variable with a value of the secret key, like `ZxPXnIEODXLRzYwCw1TgZ4osMfoKs9Zn6se6X/N0FnztfDvZbdOmYw==`"
+    },
+    {
+      "name": "method",
+      "type": "pickList",
+      "label": "Method",
+      "required": true,
+      "defaultValue": "POST",
+      "options": {
+        "OPTIONS": "OPTIONS",
+        "GET": "GET",
+        "HEAD": "HEAD",
+        "POST": "POST",
+        "PUT": "PUT",
+        "DELETE": "DELETE",
+        "TRACE": "TRACE",
+        "PATCH": "PATCH"
+      },
+      "helpMarkDown": "Select the HTTP method with which the function should be invoked."
+    },
+    {
+      "name": "headers",
+      "type": "multiLine",
+      "label": "Headers",
+      "required": false,
+      "defaultValue": "{\n\"Content-Type\":\"application/json\", \n\"PlanUrl\": \"$(system.CollectionUri)\", \n\"ProjectId\": \"$(system.TeamProjectId)\", \n\"HubName\": \"$(system.HostType)\", \n\"PlanId\": \"$(system.PlanId)\", \n\"JobId\": \"$(system.JobId)\", \n\"TimelineId\": \"$(system.TimelineId)\", \n\"TaskInstanceId\": \"$(system.TaskInstanceId)\", \n\"AuthToken\": \"$(system.AccessToken)\"\n}",
+      "properties": {
+        "resizable": true,
+        "rows": "10",
+        "maxLength": "5000"
+      },
+      "helpMarkDown": "Define a header in JSON format. The header shall be attached to the request that is sent."
+    },
+    {
+      "name": "queryParameters",
+      "type": "string",
+      "label": "Query parameters",
+      "defaultValue": "",
+      "required": false,
+      "helpMarkDown": "Query parameters string to append to the function URL. It should not start with with \"?\" nor \"&\"."
```

Comment:
nit; There is double negation in this sentence. I believe it should be "It should start neither with ? nor &." or "It should not start either with ? or &.". Also note duplicate "with".